### PR TITLE
[13.0][ADD] account_check_date

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module adds a check date on payment for check printing.

### CODE REVIEW

File: account_check_date/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+To use this module, you need to go to:
+
+* Invoicing > Vendors > Payments
```
Comment (Saran440) [inline account_check_date/readme/USAGE.rst]:
```suggestion
* Create Vendor Bills and Register Payment
* Choose payment method is check
* Select check date and register normal process
* On payment view, you see Check Date field
```

**Review by Saran440 — COMMENTED** [review]
Code Reviewed

File: account_check_date/readme/USAGE.rst (hunk 1 above)
Comment (ps-tubtim) [reply account_check_date/readme/USAGE.rst]: Edited.

**Comment by ps-tubtim** [comment]
@Saran440 Thank you for your review. I edited.

**Review by Saran440 — APPROVED** [review]
Functional Test. LGTM :+1:

**Review by victoralmau — APPROVED** [review]

**Pull request merged**